actiontext/CHANGELOG.md

```diff
@@ -7,7 +7,11 @@
 
     *Matt Swanson*, *Joel Drapper*
 
-*   Update bundled Trix version from `1.3.1` to `2.0.4`.
+*   Update bundled Trix version from `2.0.0` to `2.0.4`.
+
+    *Sarah Ridge*
+
+*   Update bundled Trix version from `1.3.1` to `2.0.0`.
 
     *Sean Doyle*
```

Comment:
```suggestion
*   Update bundled Trix version from `1.3.1` to `2.0.4`.

    *Sarah Ridge*, *Sean Doyle*
```

I would combine these CHANGELOG entries, since the `2.0.0` version hasn't ever been released

Reply:
Good call! Done!